container/rules/docker_toolchains.bzl

```diff
@@ -12,13 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Definitions of language_tool_layer and toolchain_container rules."""
+"""Definitions of language_tool_layer, debian_pkg_tar and toolchain_container rules."""
 
 load("@io_bazel_rules_docker//container:container.bzl", _container = "container")
 load("@base_images_docker//package_managers:download_pkgs.bzl", _download = "download")
 load("@base_images_docker//package_managers:install_pkgs.bzl", _install = "install")
 load("@base_images_docker//package_managers:apt_key.bzl", _key = "key")
 
+IntermediateInfo = provider(fields = [
```

Comment:
this is not a good name, what type of intermediat info are you storing, its intermediating between what and what

Reply:
Will rename it in the next patch.